src/test/java/PageModel/HomePage.java

```diff
@@ -0,0 +1,21 @@
+package PageModel;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+
+public class HomePage extends Basepage {
+    public HomePage(WebDriver givenDriver) {
+
+        super(givenDriver);
+    }
+
+    By avatarLogo = By.cssSelector(".avatar");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```